scripts/sync_s3_env.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+# Syncs two environments S3 buckets and their config files.
+# Can only go from prod -> dev <-> dev-green
+
+set -eu
+
+SCREENS_BUCKET="s3://mbta-screens/screens"
+SCREENS_JSON="s3://mbta-ctd-config/screens/screens"
+PENDING_SCREENS_JSON="s3://mbta-ctd-config/screens/pending-screens"
```

Comment:
nit: These variable names seem a bit misleading since they're more "prefixes" to an eventual full path. Maybe `SCREENS_BUCKET_PREFIX`, `SCREENS_CONFIG_PREFIX`, and `PENDING_SCREENS_PREFIX`?